src/ralph/lib/transitions/models.py

```diff
@@ -238,10 +241,9 @@ def run_field_transition(
     """
     Execute all actions assigned to the selected transition.
     """
-    if not isinstance(instances, Iterable):
+    if not isinstance(instances, list):
```

Comment:
We expect that here is a list of instances not queryset.
Because that list is modified for bulk transitions.
If this is not a list that is that we operate on one object and we can create a new one.